src/bilder/images/edxapp_v2/templates/edxapp/mitxonline/common_values.yml.tmpl

```diff
@@ -388,6 +388,11 @@ LANGUAGE_COOKIE: {{ env "ENVIRONMENT" }}-openedx-language-preference
 MIT_LEARN_AI_API_URL: https://{{ key "edxapp/learn-api-domain" }}/ai  # Added for ol_openedx_chat
 MIT_LEARN_API_BASE_URL: https://{{ key "edxapp/learn-api-domain" }}/learn  # Added for ol_openedx_chat
 MIT_LEARN_SUMMARY_FLASHCARD_URL: https://{{ key "edxapp/learn-api-domain" }}/learn/api/v1/contentfiles/  # Added for ol_openedx_chat
+MIT_LEARN_BASE_URL = "https://learn.mit.edu"
+MIT_LEARN_SUPPORT_EMAIL = "mitlearn-support@mit.edu"
+MIT_LEARN_API_BASE_URL = "https://api.learn.mit.edu"
+MIT_LEARN_LOGO = https://{{ key "edxapp/lms-domain" }}/static/mitxonline/images/mit-learn-logo.svg"
+MITXT_COURSE_KEY_FORMAT = "course-v1:mitxt"
```

Comment:
@blarghmatey We need to update these settings. They are currently set to what they should point to.